src/Core/Service/DependencyInjection/services.xml

```diff
@@ -127,13 +131,10 @@
 
         <service id="Shopware\Core\Service\Manager">
             <argument type="service" id="Shopware\Core\Framework\App\Privileges\Privileges"/>
-            <argument type="service" id="Doctrine\DBAL\Connection"/>
-        </service>
-
-        <service id="Shopware\Core\Service\Subscriber\PermissionsSubscriber">
```

Comment:
Why does this service got deleted????

Reply:
I think by mistake, since in the initial commits this this subscriber was initially removed.